### DISCUSSION TITLE
New feeds not correctly display the starred repo from people I following

### DISCUSSION THREAD

**Discussion by mikialex** [community/community#66646] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I used to check what repos are starred by the people I follow, to explore the ideas, community and ecosystem. Besides the git itself, it is literally the most important feature for me to use GitHub. 

However, with the new feeds, even it has its filter config that is exactly doing what it was able to do. I still can not see any starred repos from people I follow like before.

So I do not understand what is happening. Is this feature be totally removed(but why can I see filter options about it)? Or just a filter bug(but such a serious bug gets released and not fixed at all)?

**Comment by brendanzab** [comment]
You can revert back to the old style of feed with [this userscript](https://github.com/orgs/community/discussions/66260), but it’s not clear how long that will continue to work for. I really hope they bring back support for this use case (along with the option to default to a chronological feed).
